en/docs/develop/mi-for-vscode/best-practices.md

```diff
@@ -227,13 +227,500 @@ This guide provides best practices for designing, developing, deploying, and mai
 
 ## Migration Best Practices
 
-If you are migrating from an older version of WSO2 Integrator: MI (&lt;4.4.0), your existing configurations and mediators will continue to work in newer MI versions. Migration is optional, only perform it if you want to take advantage of newer features or performance improvements. If your current mediators meet your requirements and you are satisfied with their behavior and performance, you may continue using them without changes.
+If you are migrating from an older version of WSO2 Integrator: MI (&lt;4.4.0), your existing configurations and mediators will continue to work in newer MI versions. Migration is optional, only perform it if you want to take advantage of usability and maintainability improvements. If your current mediators meet your requirements and you are satisfied with their behavior, you may continue using them without changes.
 
-For future implementations, adopt the newer mediators and connectors (for example, HTTP Connector, ForEach, Scatter‑Gather, and the enhanced PayloadFactory) to benefit from improved performance and maintainability.
+For future implementations, adopt the newer mediators and connectors (for example, HTTP Connector, new ForEach, Scatter‑Gather, and the new PayloadFactory) to benefit from improved usability and maintainability.
 
 - The [Out Sequence]({{base_path}}/reference/mediation-sequences/#inout-sequences) in APIs and Proxies are not created by MI VS Code extension by default from version 4.4.0 onwards. Implement the necessary logic in the [In Sequence]({{base_path}}/reference/mediation-sequences/#inout-sequences) to handle outgoing messages.
+
+    ??? "OutSequence Migration Sample"
+
+        === "Legacy Configuration"
+            ```xml
+            <api context="/HelloWorld" name="HelloWorld" xmlns="http://ws.apache.org/ns/synapse">
+                <resource methods="GET">
+                    <inSequence>
+                        <payloadFactory media-type="json" template-type="default">
+                            <format>{"Hello":"World"}</format>
+                        </payloadFactory>
+                        <loopback/>
```

Comment:
It would be better if we can give a sample with send mediator. Historically, inSequence and outSequence were used to represent request flow and response flow.  So most of the exisiting code would be in this formet.

Reply:
Fixed in [9076bfc](https://github.com/wso2/docs-mi/pull/1841/commits/9076bfcb1957848ebe06dbfee8cec1e1962df1cc)